corehq/apps/data_interfaces/tasks.py

```diff
@@ -93,7 +93,13 @@ def run_case_update_rules(now=None):
                .order_by('domain'))
     for domain in domains:
         if not any_migrations_in_progress(domain) and not DISABLE_CASE_UPDATE_RULE_SCHEDULED_TASK.enabled(domain):
-            run_case_update_rules_for_domain.delay(domain, now)
+            domain_obj = Domain.get_by_name(domain)
+            if domain_obj.auto_case_update_hour is None:
+                hour_to_run = settings.RULE_UPDATE_HOUR
+            else:
+                hour_to_run = domain_obj.auto_case_update_hour
+            if hour_to_run == datetime.utcnow().hour:
```

Comment:
I thought the parameter `now` might be a celery thing that passes in the time and would be the right thing to use here instead of `utcnow`, but haven't found found proof of this in the celery docs.

Reply:
Yeah, i was hoping celery passed the runtime in as well, but I think this argument is just a way for us to manually trigger it for previous days (or now previous times).